org.lyx.LyX.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.lyx.LyX
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: lyx
+rename-desktop-file: lyx.desktop
+rename-icon: lyx
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --persist=.lyx
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-documents
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "23.08"
+    directory: texlive
+    autodelete: false
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
```

Comment:
I don't think the `sdk-extensions` is needed here (at build time)

Reply:
yes you're right.